packages/mux-uploader/src/index.ts

```diff
@@ -168,17 +177,20 @@ template.innerHTML = `
 
 <p class="upload-instruction" id="upload-instruction">Drop file to upload</p>
 
+<div class="sr-only" id="sr-only" aria-live="polite"></div>
+
 <div class=text-container>
-  <span class="status-message" id="status-message"></span>
-  <span class="retry-message" id="retry-message">Try again</span>
+  <span class="status-message" id="status-message" aria-live="polite"></span>
```

Comment:
non-blocking: I think you may want to give this a [`role="status"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role) to use it how you're intending? See how we've implemented [media-loading-indicator](https://github.com/muxinc/media-chrome/blob/main/src/js/media-loading-indicator.js#L65)